test/models/whois_record_test.rb

```diff
@@ -22,14 +22,17 @@ def test_generated_json_has_expected_values
       registrant_kind: 'priv',
       email: 'john@inbox.test',
       expire: '2010-07-05',
-      nameservers: ['ns1.bestnames.test', 'ns2.bestnames.test'],
       registrar_address: 'Main Street, New York, New York, 12345',
       dnssec_keys: [],
     }
 
     expected_partial_hash.each do |key, value|
       assert_equal(value, @record.generated_json[key])
     end
+
+    ['ns1.bestnames.test', 'ns2.bestnames.test'].each do |name_server|
```

Comment:
I think it's better to stick to `nameserver`, instead of `name_server` for consistency.

Reply:
https://www.namecheap.com/support/knowledgebase/article.aspx/9434/32/using-default-nameservers-vs-hosting-nameservers

One of the popular domain name sellers also uses single word spelling, whereas WIKI uses separate words.

Reply:
If you want to rename it to `name_server`, let's create a new branch and make it there.

Reply:
Changed to item, as this is exactly what is important here (specific item being placed in some array of item).